networkx/algorithms/operators/binary.py

```diff
@@ -45,6 +42,20 @@ def union(G, H, rename=(None, None), name=None):
     to the union graph.  If a graph attribute is present in both
     G and H the value from H is used.
 
+    Examples
+    --------
+    >>> import networkx as nx
+    >>> from networkx.algorithms.operators import *
+    >>> G = nx.Graph()
+    >>> H = nx.Graph()
+    >>> G.add_edges_from([(0, 1), (0, 2), (1, 2)])
+    >>> H.add_edges_from([(3, 4), (3, 6), (4, 5), (5, 6)])
```

Comment:
Yeah I got confused because of the docstring that mentions `Graphs G and H must be disjoint, otherwise an exception is raised.`. Maybe we should add that the graphs should be disjoint after the renaming happens.

Reply:
Yes, the docstrings should reflect the restrictions that the code needs -- :}
If the code allows duplicate nodes when renaming makes them unique, the doc_strings should say that.